Sources/Plasma/CoreLib/HeadSpin.cpp

```diff
@@ -47,6 +47,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #   include <crtdbg.h>
 #endif
 
+#if __APPLE__
```

Comment:
We can hopefully use `HS_BUILD_FOR_APPLE` anywhere that we need Apple-specific includes.

Reply:
I'll make this change too. I think this was one of the first things I did before I had a firm sense of the preprocessor macros available, and it just stuck around.